src/legacy/core_plugins/timelion/server/routes/run.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable @typescript-eslint/no-var-requires */
+import Joi from 'joi';
+import Bluebird from 'bluebird';
+import _ from 'lodash';
+import { Legacy } from 'kibana';
+// @ts-ignore
+import chainRunnerFn from '../handlers/chain_runner.js';
+const timelionDefaults = require('../lib/get_namespaced_settings')();
+
+export interface TimelionRequestQuery {
+  payload?: {
+    sheet: string[];
+    extended: {
```

Comment:
I think the `time` payload is missing here. Even if we don't use it I think we should type it somehow.

Reply:
I added time to the payload: https://github.com/elastic/kibana/pull/51788/files#diff-20042aff4267d58aaceb45fc0c1fb25bR44